addons/mail/static/src/core/common/composer.xml

```diff
@@ -100,7 +100,9 @@
                 <Picker t-props="picker"/>
             </div>
         </div>
-        <span t-if="props.composer.message" class="text-muted" t-out="CANCEL_OR_SAVE_EDIT_TEXT" t-on-click="onClickCancelOrSaveEditText"/>
```

Comment:
Can't we at least prevent default on click to avoid focusing or something like that?

Reply:
Prevent default (for focus) should be on onMouseDown apparently.

Reply:
It doesn't reach this click handler, the focusin on the composer + rendering happens before anything else

Reply:
`focusin` cannot be prevented with `mousedown` nor `click`